cluster/vagrant/provision-network-minion.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# provision-network-minion configures flannel on the minion
+function provision-network-minion {
+
+  echo "Provisioning network on minion"
```

Comment:
All of these steps run on both the master and minion, right? It seems like there's quite a bit of duplicated shell script as a result, which might be better off combined into a single function.